### DISCUSSION TITLE
Is homebrew compatible with M1 Mac?

### DISCUSSION THREAD

**Discussion by MRR71** [Homebrew/discussions#678] [Getting started]
I am trying to install lammps on my M1 Macbook Pro with homebrew. But every time I execute the $brew install lammps$ command, it shows: **bottle not available** and suggests building from source.  If I attempt to build from source, it then shows that it doesn't have access. I am wondering is it a compatibility issue with M1 mac or with Homebrew? Or something simple that I am missing here? 
Thanks, any help will be greatly appreciated.

**Comment by SMillerDev** [comment]
See earlier discussions: https://github.com/Homebrew/discussions/discussions/649 
---
> I am trying to install lammps on my M1 Macbook Pro with homebrew. But every time I execute the $brew install lammps$ command, it shows: bottle not available and suggests building from source. If I attempt to build from source, it then shows that it doesn't have access. I am wondering is it a compatibility issue with M1 mac or with Homebrew?

Anything that doesn't have a bottle is not yet compatible with M1/ARM.

**Reply by carlocab** [reply]
See also the answers you were given the last time you brought this up:

https://github.com/Homebrew/homebrew-core/issues/69976

**Reply by MRR71** [reply]
Thanks for the prompt reply. Appreciate this.
